proxy/core/src/main/resources/cloudstate-common.conf

```diff
@@ -39,6 +42,8 @@ akka {
     artery {
       enabled = on
       transport = tcp
+      // FIXME do we need to set canonical.hostname?
+      // FIXME do we need to set advanced.tcp.connection-timeout?
```

Comment:
@pvlugter @patriknw Should we set these two new settings, and to what?

Reply:
default connection-timeout is 5s, so unless you think that is wrong you should start with defaults

Reply:
if you don't set hostname it will be `InetAddress.getLocalHost.getHostAddress`, maybe good to have that as an optional environment variable here?